### DISCUSSION TITLE
如何批量下载链接的同时，自定义视频标题

### DISCUSSION THREAD

**Discussion by duomm7** [nilaoda/N_m3u8DL-CLI#541] [Q&A] [answered]
如图，获取到了一些视频的下载链接，想要批量下载的同时，自定义命名视频标题，请问各位有方法可以实现吗？
现在我都是一个一个手动下载的...
之前试过先建一个txt，每行放一个下载链接，全部下载完在改名字，也挺费时间的，求问有没有简便的方法呢？
![image](https://user-images.githubusercontent.com/34161125/141128586-e4dcc372-d86f-4e83-95c8-603ebeaee7fa.png)

**Comment by nilaoda** [comment, marked as answer]
腾讯视频每行一个txt直接拖入gui应该会自动识别标题的。

此外还有一种方法 https://nilaoda.github.io/N_m3u8DL-CLI/SimpleGUI.html

**Reply by duomm7** [reply]
谢谢回复！txt里name,url的方法我会狠狠记住的！
腾讯视频的自动识别标题偶尔会有点小彩蛋，识别到的是什么“xxxx第一期第一版/修正版/xxxx”这种类似的，应该是内部命名吧哈哈哈，所以有时候我会复制网页标题重新命名一下~
